com/win32comext/axdebug/Test/host.py

```diff
@@ -97,71 +96,25 @@ def GetPathName(self):
         try:
             return win32api.GetFullPathName(self.module.__file__), 1
         except (AttributeError, win32api.error):
-            raise Exception(scode == E_FAIL)
+            raise Exception(scode=winerror.E_FAIL)
 
     def GetFileName(self):
         # Result is a string with just the name of the document, no path information.
         trace("GetFileName")
-        return os.path.split(module.__file__)
+        return os.path.split(self.module.__file__)
 
     def NotifyChanged():
         trace("NotifyChanged")
         raise Exception(scode=winerror.E_NOTIMPL)
 
 
-def TestSmartHelper():
-    pdm = pythoncom.CoCreateInstance(
-        axdebug.CLSID_ProcessDebugManager,
-        None,
-        pythoncom.CLSCTX_ALL,
-        axdebug.IID_IProcessDebugManager,
-    )
-    app = pdm.CreateApplication()
-    app.SetName("Python Process")
-
-    pydebugger = adb.Debugger()
-
-    nodes = BuildModuleTree()
-
-    all_real_nodes = CreateDebugDocumentHelperNodes(pdm, app, nodes)
-    root = app.GetRootNode()
-    AttachParentNodes(root, nodes, all_real_nodes)
-
-    pydebugger.AttachApp(app)
-    cookie = pdm.AddApplication(app)
-    input("Waiting...")
-    ttest.test()
-
-    pdm.RemoveApplication(cookie)
-    print("Done")
-
-
-def testdumb():
-    pdm = pythoncom.CoCreateInstance(
-        axdebug.CLSID_ProcessDebugManager,
-        None,
-        pythoncom.CLSCTX_ALL,
-        axdebug.IID_IProcessDebugManager,
-    )
-    app = pdm.GetDefaultApplication()
-
-    nodes = BuildModuleTree()
-    all_real_nodes = CreateDebugDocumentHelperNodes(pdm, app, nodes)
-    AttachParentNodes(None, nodes, all_real_nodes)
-
-    parentNode = None
-    all_real_nodes = {}
-    input("Waiting...")
-    print("Done")
-
-
```

Comment:
These were commented out in the caller code (so it's unused) and contain a bunch of undefined names / unbound variables that I can't find anywhere.
If you still think it's worth keeping around, I'll comment them out instead.